### DISCUSSION TITLE
Bosean FS-5000: Support for update through USB

### DISCUSSION THREAD

**Discussion by soosp** [Gissio/radpro#171] [Ideas]
First thank you for this easy to use and function rich open source firmware! It works well on Bosean FS-5000.
My question: What do you think, is this possible to add a bootloader and update the firmware via USB without opening the enclosure?

**Comment by Hantervit** [comment]
Hello. Most likely there is no such possibility. Some users insert ST-Link directly into the dosimeter body, but it is necessary to remove a separate tapy-C port...
